client/server/boot/index.js

```diff
@@ -26,7 +26,11 @@ export default function setup() {
 	app.use( loggerMiddleware() );
 
 	if ( process.env.USE_SERVER_PROFILER === 'true' ) {
-		app.use( require( 'calypso/server/middleware/profiler' )() );
+		// TODO: Re-enable this middleware once `v8-profiler-next` adds support for Node.js 22
+		// app.use( require( 'calypso/server/middleware/profiler' )() );
+		console.warn(
+			'Server profiling is temporarily disabled until `v8-profiler-next` adds support for Node.js 22.'
```

Comment:
I'm disabling the profiler temporarily until Node 22 support is added. This is based on the [previous decision](https://github.com/Automattic/wp-calypso/pull/81798#issuecomment-1776778283), made during the previous update.